modules/psap-node-feature-discovery-using-topology-updater.adoc

```diff
@@ -8,11 +8,51 @@
 
 The Node Feature Discovery (NFD) Topology Updater is a daemon responsible for examining allocated resources on a worker node. It accounts for resources available to be allocated to new pod on a per-zone basis, where a zone can be a Non-Uniform Memory Access (NUMA) node. The NFD Topology Updater communicates the information to nfd-master, which creates a `NodeResourceTopology` custom resource corresponding to all the nodes in the cluster. One instance of the NFD Topology Updater runs on each node of the cluster.
 
-.Procedure
+To enable the Topology Updater workers in NFD, simply set the var `topologyupdater` to `true` in the NodeFeatureDiscovery CR as described in xref:../hardware_enablement/psap-node-feature-discovery-operator.adoc#using-the-node-feature-discovery-operator_node-feature-discovery-operator[Usng the Node Feature Discovery Operator].
```

Comment:
Due to our current tooling, we're currently not able to include xrefs in modules, only assemblies so we'll have to remove this link :disappointed:

Reply:
how can I point to another section of the same NFD docs?